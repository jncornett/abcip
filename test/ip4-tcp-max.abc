a(syn)
b(syn, ack)
a(ack)

# set mtu=128 and mss=500
a(ip4:max=128; tcp:ack, len=1024, max=500)
b(ack)

# change mtu=256
a(ip4:max=256; tcp:ack, len=1024 )
b(ack)

a(fin,ack)
b(fin,ack)
a(ack)

