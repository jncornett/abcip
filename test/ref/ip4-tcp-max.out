IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [S], cksum 0xb3d9 (correct), seq 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0001 0000 4006 5cbc 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0001 0000 0000 5002  ..............P.
	0x0030:  2000 b3d9 0000                           ......
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [S.], cksum 0xb3c7 (correct), seq 1, ack 2, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0001 0000 4006 5cbc 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0001 0000 0002 5012  ..............P.
	0x0030:  2000 b3c7 0000                           ......
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0xb3c8 (correct), ack 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0002 0000 4006 5cbb 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0002 0000 0002 5010  ..............P.
	0x0030:  2000 b3c8 0000                           ......
IP (tos 0x0, ttl 64, id 3, offset 0, flags [+], proto TCP (6), length 148)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], seq 1:109, ack 1, win 8192, length 108
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0094 0003 2000 4006 3c4e 0a01 0203 0a09  ......@.<N......
	0x0020:  0807 bdec 0008 0000 0002 0000 0002 5010  ..............P.
	0x0030:  2000 ece5 0000 4c6f 7265 6d20 6970 7375  ......Lorem.ipsu
	0x0040:  6d20 646f 6c6f 7220 7369 7420 616d 6574  m.dolor.sit.amet
	0x0050:  2c20 636f 6e73 6563 7465 7475 7220 6164  ,.consectetur.ad
	0x0060:  6970 6973 6369 6e67 2065 6c69 742e 2045  ipiscing.elit..E
	0x0070:  7469 616d 2073 7573 6369 7069 742c 2071  tiam.suscipit,.q
	0x0080:  7561 6d20 6174 2073 6365 6c65 7269 7371  uam.at.scelerisq
	0x0090:  7565 2075 6c74 7269 6369 6573 2c20 6e69  ue.ultricies,.ni
	0x00a0:  7369                                     si
IP (tos 0x0, ttl 64, id 3, offset 128, flags [+], proto TCP (6), length 148)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0094 0003 2010 4006 3c3e 0a01 0203 0a09  ......@.<>......
	0x0020:  0807 206e 756c 6c61 2066 6572 6d65 6e74  ...nulla.ferment
	0x0030:  756d 206f 7263 692c 2061 2069 6d70 6572  um.orci,.a.imper
	0x0040:  6469 6574 206c 6163 7573 2065 6e69 6d20  diet.lacus.enim.
	0x0050:  6120 6665 6c69 732e 2043 756d 2073 6f63  a.felis..Cum.soc
	0x0060:  6969 7320 6e61 746f 7175 6520 7065 6e61  iis.natoque.pena
	0x0070:  7469 6275 7320 6574 206d 6167 6e69 7320  tibus.et.magnis.
	0x0080:  6469 7320 7061 7274 7572 6965 6e74 206d  dis.parturient.m
	0x0090:  6f6e 7465 732c 206e 6173 6365 7475 7220  ontes,.nascetur.
	0x00a0:  7269                                     ri
IP (tos 0x0, ttl 64, id 3, offset 256, flags [+], proto TCP (6), length 148)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0094 0003 2020 4006 3c2e 0a01 0203 0a09  ......@.<.......
	0x0020:  0807 6469 6375 6c75 7320 6d75 732e 204e  ..diculus.mus..N
	0x0030:  756c 6c61 6d20 6461 7069 6275 7320 7269  ullam.dapibus.ri
	0x0040:  7375 7320 696e 2064 6f6c 6f72 2069 6163  sus.in.dolor.iac
	0x0050:  756c 6973 2071 7569 7320 6c61 6369 6e69  ulis.quis.lacini
	0x0060:  6120 6573 7420 7065 6c6c 656e 7465 7371  a.est.pellentesq
	0x0070:  7565 2e20 4d61 6563 656e 6173 2075 7420  ue..Maecenas.ut.
	0x0080:  6665 6c69 7320 6573 742c 2073 6564 2069  felis.est,.sed.i
	0x0090:  6e74 6572 6475 6d20 6c6f 7265 6d2e 2043  nterdum.lorem..C
	0x00a0:  7261                                     ra
IP (tos 0x0, ttl 64, id 3, offset 384, flags [+], proto TCP (6), length 148)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0094 0003 2030 4006 3c1e 0a01 0203 0a09  .....0@.<.......
	0x0020:  0807 7320 6175 6374 6f72 2c20 6970 7375  ..s.auctor,.ipsu
	0x0030:  6d20 7669 7461 6520 7665 6e65 6e61 7469  m.vitae.venenati
	0x0040:  7320 6661 7563 6962 7573 2c20 6e75 6e63  s.faucibus,.nunc
	0x0050:  206d 6173 7361 2066 6572 6d65 6e74 756d  .massa.fermentum
	0x0060:  2061 7567 7565 2c20 7669 7461 6520 756c  .augue,.vitae.ul
	0x0070:  6c61 6d63 6f72 7065 7220 7075 7275 7320  lamcorper.purus.
	0x0080:  6665 6c69 7320 7175 6973 2061 6e74 652e  felis.quis.ante.
	0x0090:  2046 7573 6365 2066 6575 6769 6174 2065  .Fusce.feugiat.e
	0x00a0:  6e69                                     ni
IP (tos 0x0, ttl 64, id 3, offset 512, flags [none], proto TCP (6), length 28)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  001c 0003 0040 4006 5c86 0a01 0203 0a09  .....@@.\.......
	0x0020:  0807 6d20 7574 206e 756e                 ..m.ut.nun
IP (tos 0x0, ttl 64, id 4, offset 0, flags [+], proto TCP (6), length 148)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], seq 501:609, ack 1, win 8192, length 108
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0094 0004 2000 4006 3c4d 0a01 0203 0a09  ......@.<M......
	0x0020:  0807 bdec 0008 0000 01f6 0000 0002 5010  ..............P.
	0x0030:  2000 5bb9 0000 6320 636f 6e73 6563 7465  ..[...c.consecte
	0x0040:  7475 7220 6964 2075 6c74 7269 6369 6573  tur.id.ultricies
	0x0050:  2074 6f72 746f 7220 6c6f 626f 7274 6973  .tortor.lobortis
	0x0060:  2e20 4d61 6563 656e 6173 2069 6e20 6f72  ..Maecenas.in.or
	0x0070:  6369 206d 6574 7573 2c20 6e6f 6e20 696e  ci.metus,.non.in
	0x0080:  7465 7264 756d 206e 6571 7565 2e20 5065  terdum.neque..Pe
	0x0090:  6c6c 656e 7465 7371 7565 2076 756c 7075  llentesque.vulpu
	0x00a0:  7461                                     ta
IP (tos 0x0, ttl 64, id 4, offset 128, flags [+], proto TCP (6), length 148)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0094 0004 2010 4006 3c3d 0a01 0203 0a09  ......@.<=......
	0x0020:  0807 7465 2061 6e74 6520 696e 2072 6973  ..te.ante.in.ris
	0x0030:  7573 2075 6c6c 616d 636f 7270 6572 2072  us.ullamcorper.r
	0x0040:  7574 7275 6d2e 2020 4675 7363 6520 7665  utrum...Fusce.ve
	0x0050:  6c20 6461 7069 6275 7320 6c65 6f2e 2049  l.dapibus.leo..I
	0x0060:  6e20 7365 6420 6c61 6375 7320 6e65 6320  n.sed.lacus.nec.
	0x0070:  6c61 6375 7320 7469 6e63 6964 756e 7420  lacus.tincidunt.
	0x0080:  696d 7065 7264 6965 7420 6120 7665 6c20  imperdiet.a.vel.
	0x0090:  6d61 676e 612e 2053 7573 7065 6e64 6973  magna..Suspendis
	0x00a0:  7365                                     se
IP (tos 0x0, ttl 64, id 4, offset 256, flags [+], proto TCP (6), length 148)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0094 0004 2020 4006 3c2d 0a01 0203 0a09  ......@.<-......
	0x0020:  0807 206c 7563 7475 7320 636f 6e67 7565  ...luctus.congue
	0x0030:  2069 6d70 6572 6469 6574 2e20 446f 6e65  .imperdiet..Done
	0x0040:  6320 7365 6420 7361 6769 7474 6973 206c  c.sed.sagittis.l
	0x0050:  6f72 656d 2e20 496e 2068 6163 2068 6162  orem..In.hac.hab
	0x0060:  6974 6173 7365 2070 6c61 7465 6120 6469  itasse.platea.di
	0x0070:  6374 756d 7374 2e20 446f 6e65 6320 736f  ctumst..Donec.so
	0x0080:  6c6c 6963 6974 7564 696e 2061 6e74 6520  llicitudin.ante.
	0x0090:  6574 2061 6e74 6520 7469 6e63 6964 756e  et.ante.tincidun
	0x00a0:  7420                                     t.
IP (tos 0x0, ttl 64, id 4, offset 384, flags [+], proto TCP (6), length 148)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0094 0004 2030 4006 3c1d 0a01 0203 0a09  .....0@.<.......
	0x0020:  0807 7574 2070 6f72 7461 2065 6c69 7420  ..ut.porta.elit.
	0x0030:  7268 6f6e 6375 732e 2050 6861 7365 6c6c  rhoncus..Phasell
	0x0040:  7573 2072 6973 7573 206e 756e 632c 2066  us.risus.nunc,.f
	0x0050:  7269 6e67 696c 6c61 2061 2074 696e 6369  ringilla.a.tinci
	0x0060:  6475 6e74 2069 642c 206d 616c 6573 7561  dunt.id,.malesua
	0x0070:  6461 2061 7420 7361 7069 656e 2e20 4372  da.at.sapien..Cr
	0x0080:  6173 2076 6974 6165 206d 6574 7573 206a  as.vitae.metus.j
	0x0090:  7573 746f 2c20 6964 2061 7563 746f 7220  usto,.id.auctor.
	0x00a0:  6d65                                     me
IP (tos 0x0, ttl 64, id 4, offset 512, flags [none], proto TCP (6), length 28)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  001c 0004 0040 4006 5c85 0a01 0203 0a09  .....@@.\.......
	0x0020:  0807 7475 732e 2043 7572                 ..tus..Cur
IP (tos 0x0, ttl 64, id 5, offset 0, flags [none], proto TCP (6), length 64)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0xe807 (correct), seq 1001:1025, ack 1, win 8192, length 24
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0040 0005 0000 4006 5ca0 0a01 0203 0a09  .@....@.\.......
	0x0020:  0807 bdec 0008 0000 03ea 0000 0002 5010  ..............P.
	0x0030:  2000 e807 0000 6162 6974 7572 2073 7573  ......abitur.sus
	0x0040:  6369 7069 7420 6661 6369 6c69 7369       cipit.facilisi
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [.], cksum 0xafc8 (correct), ack 1025, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0002 0000 4006 5cbb 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0002 0000 0402 5010  ..............P.
	0x0030:  2000 afc8 0000                           ......
IP (tos 0x0, ttl 64, id 6, offset 0, flags [+], proto TCP (6), length 276)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], seq 1025:1261, ack 1, win 8192, length 236
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0114 0006 2000 4006 3bcb 0a01 0203 0a09  ......@.;.......
	0x0020:  0807 bdec 0008 0000 0402 0000 0002 5010  ..............P.
	0x0030:  2000 81cd 0000 7320 6665 6c69 7320 7669  ......s.felis.vi
	0x0040:  7461 6520 7068 6172 6574 7261 2e20 5574  tae.pharetra..Ut
	0x0050:  2065 7520 6e69 7369 206e 6973 692e 2043  .eu.nisi.nisi..C
	0x0060:  7261 7320 636f 6e64 696d 656e 7475 6d20  ras.condimentum.
	0x0070:  6575 6973 6d6f 6420 6d61 7373 6120 6575  euismod.massa.eu
	0x0080:  2074 7269 7374 6971 7565 2e20 446f 6e65  .tristique..Done
	0x0090:  6320 6572 6f73 206e 6973 6c2c 2073 656d  c.eros.nisl,.sem
	0x00a0:  7065 7220 6574 206c 6163 696e 6961 2061  per.et.lacinia.a
	0x00b0:  2c20 7465 6d70 7573 2069 6420 6469 616d  ,.tempus.id.diam
	0x00c0:  2e20 4372 6173 206e 6f6e 2074 656c 6c75  ..Cras.non.tellu
	0x00d0:  7320 6e6f 6e20 7365 6d20 636f 6e67 7565  s.non.sem.congue
	0x00e0:  206d 6f6c 6c69 7320 6e6f 6e20 7574 206d  .mollis.non.ut.m
	0x00f0:  692e 2044 7569 7320 6469 616d 2066 656c  i..Duis.diam.fel
	0x0100:  6973 2c20 6375 7273 7573 206e 6f6e 2073  is,.cursus.non.s
	0x0110:  7573 6369 7069 7420 6e6f 6e2c 206d 6f6c  uscipit.non,.mol
	0x0120:  6573                                     es
IP (tos 0x0, ttl 64, id 6, offset 256, flags [+], proto TCP (6), length 276)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0114 0006 2020 4006 3bab 0a01 0203 0a09  ......@.;.......
	0x0020:  0807 7469 6520 7669 7461 6520 7465 6c6c  ..tie.vitae.tell
	0x0030:  7573 2e20 446f 6e65 6320 6661 6369 6c69  us..Donec.facili
	0x0040:  7369 7320 6475 6920 6574 2065 726f 7320  sis.dui.et.eros.
	0x0050:  706f 7274 7469 746f 7220 636f 6e73 6571  porttitor.conseq
	0x0060:  7561 742e 2020 5574 2061 2062 6962 656e  uat...Ut.a.biben
	0x0070:  6475 6d20 6c69 6775 6c61 2e20 5072 6f69  dum.ligula..Proi
	0x0080:  6e20 6575 6973 6d6f 6420 706f 7274 6120  n.euismod.porta.
	0x0090:  6c65 6f20 6120 6469 6374 756d 2e20 4165  leo.a.dictum..Ae
	0x00a0:  6e65 616e 2071 7569 7320 6970 7375 6d20  nean.quis.ipsum.
	0x00b0:  6d61 7572 6973 2c20 6575 2073 6167 6974  mauris,.eu.sagit
	0x00c0:  7469 7320 7361 7069 656e 2e20 4165 6e65  tis.sapien..Aene
	0x00d0:  616e 206e 6962 6820 6d65 7475 732c 206c  an.nibh.metus,.l
	0x00e0:  6f62 6f72 7469 7320 7365 6420 6665 7567  obortis.sed.feug
	0x00f0:  6961 7420 7574 2c20 6665 726d 656e 7475  iat.ut,.fermentu
	0x0100:  6d20 7669 7461 6520 6e69 6268 2e20 4372  m.vitae.nibh..Cr
	0x0110:  6173 2073 6365 6c65 7269 7371 7565 2c20  as.scelerisque,.
	0x0120:  6c69                                     li
IP (tos 0x0, ttl 64, id 6, offset 512, flags [+], proto TCP (6), length 276)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0114 0006 2040 4006 3b8b 0a01 0203 0a09  .....@@.;.......
	0x0020:  0807 6775 6c61 2065 7420 616c 6971 7565  ..gula.et.alique
	0x0030:  7420 7665 7374 6962 756c 756d 2c20 6572  t.vestibulum,.er
	0x0040:  6174 206d 6173 7361 2074 656d 7075 7320  at.massa.tempus.
	0x0050:  6d69 2c20 6163 2070 6f72 7474 6974 6f72  mi,.ac.porttitor
	0x0060:  2074 7572 7069 7320 6d61 676e 6120 7665  .turpis.magna.ve
	0x0070:  6c20 7572 6e61 2e20 4e75 6e63 2069 6e20  l.urna..Nunc.in.
	0x0080:  6c69 6265 726f 2061 2061 7567 7565 2076  libero.a.augue.v
	0x0090:  6573 7469 6275 6c75 6d20 6163 6375 6d73  estibulum.accums
	0x00a0:  616e 2061 7420 7574 2064 6961 6d2e 2043  an.at.ut.diam..C
	0x00b0:  7261 7320 6469 676e 6973 7369 6d20 6661  ras.dignissim.fa
	0x00c0:  6369 6c69 7369 7320 7269 7375 732c 2069  cilisis.risus,.i
	0x00d0:  6e20 6469 6374 756d 2066 656c 6973 2075  n.dictum.felis.u
	0x00e0:  6c6c 616d 636f 7270 6572 2061 632e 2053  llamcorper.ac..S
	0x00f0:  6564 2068 656e 6472 6572 6974 206c 6163  ed.hendrerit.lac
	0x0100:  7573 2076 6974 6165 206d 6175 7269 7320  us.vitae.mauris.
	0x0110:  636f 6e73 6571 7561 7420 6e6f 6e20 6c61  consequat.non.la
	0x0120:  6f72                                     or
IP (tos 0x0, ttl 64, id 6, offset 768, flags [+], proto TCP (6), length 276)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0114 0006 2060 4006 3b6b 0a01 0203 0a09  .....`@.;k......
	0x0020:  0807 6565 7420 6d65 7475 7320 636f 6e67  ..eet.metus.cong
	0x0030:  7565 2e20 4e75 6c6c 6120 6661 6369 6c69  ue..Nulla.facili
	0x0040:  7369 2e20 4e61 6d20 7574 2064 6f6c 6f72  si..Nam.ut.dolor
	0x0050:  2065 6c69 742c 2065 7420 6772 6176 6964  .elit,.et.gravid
	0x0060:  6120 6e69 7369 2e20 5068 6173 656c 6c75  a.nisi..Phasellu
	0x0070:  7320 6961 6375 6c69 7320 756c 7472 6963  s.iaculis.ultric
	0x0080:  6573 2070 6f72 7461 2e20 2043 756d 2073  es.porta...Cum.s
	0x0090:  6f63 6969 7320 6e61 746f 7175 6520 7065  ociis.natoque.pe
	0x00a0:  6e61 7469 6275 7320 6574 206d 6167 6e69  natibus.et.magni
	0x00b0:  7320 6469 7320 7061 7274 7572 6965 6e74  s.dis.parturient
	0x00c0:  206d 6f6e 7465 732c 206e 6173 6365 7475  .montes,.nascetu
	0x00d0:  7220 7269 6469 6375 6c75 7320 6d75 732e  r.ridiculus.mus.
	0x00e0:  2041 656e 6561 6e20 6175 6775 6520 7475  .Aenean.augue.tu
	0x00f0:  7270 6973 2c20 6d61 6c65 7375 6164 6120  rpis,.malesuada.
	0x0100:  6163 2076 6f6c 7574 7061 7420 6574 2c20  ac.volutpat.et,.
	0x0110:  616c 6971 7561 6d20 6120 6d61 7373 612e  aliquam.a.massa.
	0x0120:  2050                                     .P
IP (tos 0x0, ttl 64, id 6, offset 1024, flags [none], proto TCP (6), length 40)
    10.1.2.3 > 10.9.8.7: tcp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0006 0080 4006 5c37 0a01 0203 0a09  .(....@.\7......
	0x0020:  0807 726f 696e 206e 6f6e 206d 6173 7361  ..roin.non.massa
	0x0030:  206c 6962 6572                           .liber
IP (tos 0x0, ttl 64, id 3, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [.], cksum 0xabc8 (correct), ack 2049, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0003 0000 4006 5cba 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0002 0000 0802 5010  ..............P.
	0x0030:  2000 abc8 0000                           ......
IP (tos 0x0, ttl 64, id 7, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [F.], cksum 0xabc7 (correct), seq 2049, ack 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0007 0000 4006 5cb6 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0802 0000 0002 5011  ..............P.
	0x0030:  2000 abc7 0000                           ......
IP (tos 0x0, ttl 64, id 4, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [F.], cksum 0xabc6 (correct), seq 1, ack 2050, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0004 0000 4006 5cb9 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0002 0000 0803 5011  ..............P.
	0x0030:  2000 abc6 0000                           ......
IP (tos 0x0, ttl 64, id 8, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0xabc6 (correct), ack 2, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0008 0000 4006 5cb5 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0803 0000 0003 5010  ..............P.
	0x0030:  2000 abc6 0000                           ......
